src/container/srv_target.c

```diff
@@ -1020,13 +891,10 @@ cont_close_one_rec(struct cont_tgt_close_rec *rec)
 	cont_stop_agg_ult(hdl->sch_cont);
 
 	dtx_batched_commit_deregister(hdl);
-	if (!hdl->sch_deleted) {
-		cont_hdl_delete(&tls->dt_cont_hdl_hash, hdl);
```

Comment:
Why you removed the cont_hdl_delete() for cont_close_one_rec()? how the container handle can be removed from the cache?

Reply:
oops, typo.

Reply:
Actually it's not typo, the cont_hdl_delete() is called in following ds_cont_local_close(). I think the reason you introduced 'sch_deleted' is that above 'dtx_batched_commit_deregister()' may yield, so I changed to call ds_cont_local_close() here so that if the handle were deleted by somebody else while yielding, we just do nothing here.

Reply:
ds_cont_local_close() is only used for rebuild case.